src/schema/fair.ts

```diff
@@ -230,6 +236,9 @@ const FairType = new GraphQLObjectType({
     tagline: {
       type: GraphQLString,
     },
+    tickets_link: {
```

Comment:
On a passthrough field like this, do we want to camelCase?

Reply:
Yup, every new field should be camelCase.

Reply:
@alloy - there's no way to rename a field with attaching a resolver, right?

Reply:
Nope, gotta do the `({ snake_case }) => snake_case` dance I'm afraid